cocotb/config.py

```diff
@@ -97,6 +103,59 @@ def help_vars_text():
     return helpmsg
 
 
+def lib_name(interface, simulator):
```

Comment:
```suggestion
def lib_name(interface: str, simulator: str) : str:
```

Reply:
Why add annotations here? This is an internal function and we aren't using mypy.

Reply:
Annotations are documentation too, and documentation isn't just for users :)